modules/module-mysql/src/replication/zongji/zongji.d.ts

```diff
@@ -0,0 +1,119 @@
+declare module '@powersync/mysql-zongji' {
```

Comment:
Should we move this to our fork package?

Reply:
I would be happy to - just not entirely sure what configuration is involved for that 😅 
I'm assuming this entails exporting it as the library's types or something?

Reply:
This file could be added to that package. It would require some minor modifications, like removing the `declare module ...` bit and exporting the individual declarations. We could then add an entry for `"types": "types/index.d.ts` to the package.json file. 
This is not a blocker for now though.